openmetadata-ui/src/main/resources/ui/src/components/Modals/EntityNameModal/EntityNameModal.component.tsx

```diff
@@ -28,14 +28,14 @@ const EntityNameModal: React.FC<EntityNameModalProps> = ({
 }) => {
   const { t } = useTranslation();
   const [form] = Form.useForm<{ name: string; displayName: string }>();
+  const [isLoading, setIsLoading] = useState(false);
 
   const handleSave = async (obj: { name: string; displayName: string }) => {
-    try {
-      await form.validateFields();
-      onSave(obj);
-    } catch (_) {
-      // Nothing here
-    }
+    setIsLoading(true);
+    await form.validateFields();
+    // Error must be handled by the parent component
+    await onSave(obj);
+    setIsLoading(false);
```

Comment:
Shouldn't this loading be set to false in catch ?

Reply:
As the error will be handled by the parent component, we don't need a catch block here!